version.go

```diff
@@ -1,3 +1,3 @@
 package beeline
 
-const version = "0.1.1"
+const version = "0.2.0"
```

Comment:
:semver-sideeye: there are API breaks.  folks are using `vgo` with semver now, right?  probably want to bump this to 1.0.0.

Reply:
it's pre-1.0, I'm allowed "beta" leeway. I'm not sure I'm quite comfortable calling this 1.0; if anything the previous one should be rebranded 1.0 and this 2.0.

Reply:
yep, I am not quite up on my semver spec reading :)

https://semver.org/#spec-item-4

Reply:
Though it is an interesting question - when should it be considered 1.0.0?